### PR TITLE
Update change log for 4.18.0-beta release

### PR DESCRIPTION
**Pull request by xinlian12**
#### New Features
* Integrated cosmos diagnostics with open telemetry tracer.
* Added `BulkExecutionOptions` and `BulkExecutionThresholds`.

#### Key Bug Fixes
* Added reactor netty timeline to query plan calls.
* Fixed serialization warning on clientSideRequestDiagnostics.
* Fixed an issue when `IdleEndpointTimeout` is 0.
* Added retry for `PrematureCloseException`.
* Fixed a hang issue in bulk executor.
* Fixed an issue which preventing recovery from 410/0 after split.
* Added TransportRequestChannelAcquisitionContext in cosmos diagnostics.

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Review by simplynaveen20 — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 ## Release History
 
-## 4.18.0-beta.1 (Unreleased)
+### 4.18.0-beta.1 (2021-08-22)
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:3]: The date is incorrect. It should be 
`2021-08-11`

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md]: oops lol, updated

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**